modules/apps/29-fee/keeper/escrow.go

```diff
@@ -108,3 +108,40 @@ func (k Keeper) DistributeFeeTimeout(ctx sdk.Context, refundAcc, timeoutRelayer
 
 	return nil
 }
+
+func (k Keeper) RefundFeesOnChannel(ctx sdk.Context, portID, channelID string) error {
```

Comment:
nit: the body of this function is really similar to the code in DistributeFee in escrow.go, maybe a way to extract the redundant code to a helper function